### PR TITLE
Social Connections: Fix Instagram issues

### PR DESCRIPTION
**Pull request by gmjuhasz**
This change fixes up some issue we have on the connections screen

## Proposed Changes

* Remove video reels from description as its not developed yet
* Adds the new logo to Mastodon
* Adds a default profile picture for Instagram and Mastodon if there is none present.

## Testing Instructions

* Open up calypso live for your site, which has the Mastodon blog sticker and newest version of Jetpack.
* Check the connections screen, it should look like this:

CleanShot 2023-06-08 at 08 08 22 png

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/marketing/connections/service-description.jsx [hunk 1]
```diff
@@ -40,8 +40,8 @@ class SharingServiceDescription extends Component {
 			instagram_business: function () {
 				if ( this.props.numberOfConnections > 0 ) {
 					return this.props.translate(
-						'Sharing photos and video reels to your Instagram account.',
-						'Sharing photos and video reels to your Instagram accounts.',
+						'Sharing photos to your Instagram account.',
```
Comment (spsiddarthan) [inline client/my-sites/marketing/connections/service-description.jsx:43]: These should be 'Share photos to your Instagram account.'

**Review by spsiddarthan — CHANGES_REQUESTED** [review]
This looks good, just made a suggestion wrt copy. Also, looks like a required check is failing.

File: client/my-sites/marketing/connections/service-description.jsx (hunk 1 above)
Comment (gmjuhasz) [reply client/my-sites/marketing/connections/service-description.jsx:43]: I think no, [those are here](https://github.com/Automattic/wp-calypso/pull/77929/files/9cd2f3a3ad6965eae4648acbd3f8ab887fcbdd61#diff-039afe5cb9721f185926d23e7b755e388fcbe7f24f624e5f897cd1645b935d68L56). We say "Share photos" while there is no connected account, and say "Sharing photos.." when they have at least one connection. At least that's how the other services work. What do you think?

Comment (spsiddarthan) [reply client/my-sites/marketing/connections/service-description.jsx:43]: Thanks for explaining it, Geri! You're right, that's how the other services work as well.

**Review by spsiddarthan — APPROVED** [review]

**Review by spsiddarthan — APPROVED** [review]

**Pull request merged**